kie-aries-blueprint/src/main/resources/org/kie/aries/blueprint/kie-aries-blueprint.xsd

```diff
@@ -97,6 +97,14 @@
     </xsd:complexType>
   </xsd:element>
 
+  <xsd:element name="import">
+    <xsd:complexType>
+      <xsd:attribute name="releaseId-ref" use="optional" type="xsd:string"/>
```

Comment:
It's optional and in that case it means that you want to import the classpath kiecontainer.